vuepress/docs/next/docs/releases/README.md

```diff
@@ -1,7 +1,139 @@
+
+### App Builder
+
+- App Builder is now extensible via MFEs loaded from bundles:
+   - Provides App Builder with alternative core components
+   - Introduces new screens accessible from a menu dedicated to EPCs
+- The same SEO-friendly code can be used for different languages on the same page
+- CSS/text files can be viewed or edited in the App Builder file browser
+- Renamed Entando Component Repository (ECR) to Local Hub
+- Improved error handling when uninstalling a bundle
+- Fixed the clone function for UX fragments
+- Minor adjustments to support docker-based bundles 
+- Minor fixes related to pagination and usability
+- Invoking APIs now sends the authentication header (instead of explicitly sending authentication cookies)
+- Added a global object `window.entando` to expose global information (e.g. lang, userPermissions, router) used by MFEs to interact with App Builder
+
+### AppEngine
+
+- Enabled Keycloak authentication for additional core APIs
+- Optimized startup procedure in cases where an automatic backup restore is implied
+- All APIs now authenticate exclusively through HTTP headers and do not require cookies
+- Fixed session invalidation issues
+
+### Entando CLI (ent)
+
+- Dropped support for Entando <= 6.3.0 and Kubernetes <= 1.18
+- [Entando Bundle CLI](#entando-bundle-cli) (`ent bundle` tool) added to ent
+- `check-env` updated to automatically install additional dependencies from GitHub packages
+- `get-plugin-id` deprecated in favor of `get-plugin-code`
+- Minor `ent pkg` fixes
+- Added [crane](https://github.com/google/go-containerregistry/blob/main/cmd/crane/doc/crane.md) as an external package
+- Added shortcuts for running internal k9s, crane, jq and fzf (e.g. `ent crane`)
+- `ent attach-kubectx` now supports `--current` to attach the current kubectl context (e.g. following an `oc login`)
+- Modified `get-bundle-id` to support docker-based bundles
+- Fixes to `ent help` and logging
+- `ent check-env` now shows a spinner (can be disabled with `--verbose`) instead of printing the full installation output of npm dependencies 
+- `ent prj` now supports `--from-hub` initialization
+- New variables with overridable defaults: `ENTANDO_CLI_DEFAULT_DOCKER_REGISTRY` and `ENTANDO_CLI_DEFAULT_HUB`
+- Implemented `ent config --effective` to show the effective configuration (composition of hard defaults, distribution defaults and profile configuration)
+- Now jq is always required by `check-env` (both runtime and develop)
+- Experimental support for an enhanced selection menu based on fzf (`ent pkg get fzf`, `ent config --set FZF_SELECT true`)
+- Workarounds to mitigate npm bugs in the update process
+- `ent-diag` now extracts additional details
+
+### Entando Bundle CLI
+
+- New CLI module "ent bundle" to manage docker-based bundles
+- Support for the following workflows:
+   - From scratch: User can create a new bundle project and then add components from existing projects or using external tools
```

Comment:
using external tools >> use external 
to match add components and use external...

Reply:
disagree. it reads "add components...using external tools"